client/layout/masterbar/logged-out.jsx

```diff
@@ -10,17 +10,23 @@ import Masterbar from './masterbar';
 		</Item>
+		<div className="masterbar__login-links">
+			<Item url={ config( 'signup_url' ) }>
+				{ translate( 'Sign Up', { context: 'Toolbar, must be shorter than ~12 chars' } ) }
```

Comment:
Super-nitpick here -- IIRC, `context` becomes a differentiating feature to the translation string, and should hence be sort of short, expressive, and easily reproducible by others. So I think it should be just something like `toolbar` here, while the note on length -- which is a nudge directed to the translator -- should go to `comment`. (Same on L26)